src/style.css

```diff
@@ -1,3 +1,97 @@
-body {
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: 'Roboto', 'Arial', sans-serif;`.